include/Enums.h

```diff
@@ -298,6 +299,7 @@ static const std::string TurbulenceModelConstantNames[] = {
   "ams_peclet_offset",
   "ams_peclet_slope",
   "ams_peclet_scale",
+  "avgTimeScaleCoeff",
```

Comment:
Let's use `avgTimeCoeff` instead to be consistent with my other implementation

Reply:
Done.